server/protocols/webadmin/webadmin-data/src/test/java/org/apache/james/webadmin/routes/UserRoutesTest.java

```diff
@@ -616,6 +618,38 @@ void putShouldReturnBadRequestWhenUsernameDoesNotHaveDomainPart() {
                 .body("details", is("Given Username needs to contain a @domainpart"));
         }
 
+        @Test
+        void putWithDomainPartInUsernameWithExistingUsernameAndNonForceParamShouldNotBeAllowed() {
+            with()
+                    .body("{\"password\":\"password\"}")
+                    .put(USERNAME_WITH_DOMAIN.asString());
+
+            with()
+                    .body("{\"password\":\"password\"}")
+                    .when()
```

Comment:
We do generaly rely on specific indentation rules to make these calls more readable:

```
given()
    .body(...)
.when() // at the same level than given
    .get(...)
.then()  // at the same level than given
    .assertions();
```